src/ecuacion/PromedioEstudiante.java

```diff
@@ -0,0 +1,81 @@
+package ecuacion;
+import java.util.Scanner;
+import java.time.LocalDate;
+
+public class PromedioEstudiante {
+	
+	public static void main(String[] args) {
+		Scanner scan = new Scanner(System.in);
+		//Ingreso de datos de estudiante
+		System.out.print("Ingrese el Nombre del estudiante: ");
+		String nameStudent = scan.nextLine();
+		System.out.print("Ingrese la Cedula del estudiante: ");
+		String dniStudent = scan.nextLine();
+		System.out.println("Ingrese la fecha de nacimiento del estudiante del estudiante: ");
+		System.out.print("Ingrese el año: ");
+		int yearBorn= scan.nextInt();
+		System.out.print("Ingrese el mes: ");
+		int monthBorn= scan.nextInt();
+		System.out.print("Ingrese el dia: ");
+		int dayBorn= scan.nextInt();
+		
+		//fecha actual en años, meses y dias
+		LocalDate today = LocalDate.now();
+		int yearDate = today.getYear();
+		int monthDate = today.getMonthValue();
+		int dayDate = today.getDayOfMonth();
+		int edad = 0;
+		
+		//Calculo de la edad
+		if(yearDate > yearBorn) {
```

Comment:
existe una forma mas eficaz, usando la clase Period. esta clase puede calcular el tiempo entre 2 fechas, something like this.

```
Period period = Period.between(birthdate, today);

// y asi obteneis los años entre una fecha y otra. 
int years = period.getYears();
```

Tambien podeis obtener los días o los meses con

```
int years = period.getDays();
int years = period.getMonths();
```

Reply:
Voy a aplicar esta a ver, gracias por el dato

Reply:
Marico esta vaina la da con mucha precision gracias

Reply:
Done

Reply:
Done